clusterman/signals/pending_pods_signal.py

```diff
@@ -58,7 +58,9 @@ def evaluate(
         retry_on_broken_pipe: bool = True,
     ) -> Union[SignalResourceRequest, List[KubernetesPod]]:
         allocated_resources = self.cluster_connector.get_cluster_allocated_resources()
-        pending_pods = self.cluster_connector.get_unschedulable_pods()
+        pending_pods = self.cluster_connector.get_unschedulable_pods(
+            detect_reason=self.parameters.get("detect_reason", True)
```

Comment:
I'm curious that if I add "detect_reason: false" to parameter in the config , this line will convert it to bool?!

Reply:
Or should I convert it to bool?